en/flight_controller/airlink.md

```diff
@@ -0,0 +1,1003 @@
+:::
+
+[AIRLink](https://sky-drones.com/airlink>) stands for Artificial Intelligence & Remote Link. The unit includes a cutting-edge drone autopilot, AI mission computer and LTE/5Gs connectivity unit. AIRLink helps to reduce the time to market for new drone manufacturers from years and months down to weeks.
+
+![AIRLink](../../assets/flight_controller/airlink/airlink-main.jpg)
+
+:::note
+This flight controller is [manufacturer supported](../flight_controller/autopilot_manufacturer_supported.md).
+:::
+
+AIRLink has two computers and integrated LTE Module: 
+
+   - The flight control computer (autopilot) has a triple-redundant vibration-dampened and temperature-stabilized IMU. 
+   - The powerful AI mission computer enables advanced drone software features like computer vision and obstacle avoidance, digital HD video streaming, and payload data streaming.
+   - LTE/5G and WiFi connectivity modules provide permanent broadband internet connection which is enabler for remote workflows.
+
+
+![Three in one](../../assets/flight_controller/airlink/airlink-3in1.png)
+
+## Feature highlights
+
+@[youtube](https://youtu.be/VcBx9DLPN54)
+
+## Specifications
+
+- **Sensors**
+
+   - 3x Accelerometers, 3x Gyroscopes, 3x Magnetometers, 3x Pressure sensorss
+   - GNSS, Rangefinders, Lidars, Optical Flow, Cameras
+   - 3x-redundant IMU
+   - Vibration dampening
+   - Temperature stabilization
+
+- **Flight Controller**
+
+   - STM32F7, ARM Cortex M7 with FPU, 216 MHz, 2MB Flash, 512 kB RAM
+   - STM32F1, I/O co-processor
+   - Ethernet, 10/100 Mbps
+   - LAN with AI Mission Computer
+   - 8x UARTs: Telemetry 1, Telemetry 2 (AI Mission Computer), Telemetry 3, GPS 1, GPS 2, Extra UART, Serial Debug Console, IO
+   - 2x CAN: CAN1, CAN2
+   - USB with MAVLink
+   - Serial console for debugging   
+   - RC Input, SBUS input, RSSI input, PPM input
+   - 16x PWM servo outputs (8 from IO, 8 from FMU)
+   - 3x I2C ports
+   - High-powered piezo buzzer driver
+   - High-power RGB LED
+   - Safety switch / LED option
+
+- **AI Mission Computer**
+
+   - 6-Core CPU: Dual-Core Cortex-A72 + Quad-Core Cortex-A53
+   - GPU Mali-T864, OpenGL ES1.1/2.0/3.0/3.1
+   - VPU with 4K VP8/9, 4K 10bits H265/H264 60fps Decoding
+   - Remote power control, software reset, power down, RTC Wake-Up, sleep mode
+   - RAM Dual-Channel 4GB LPDDR4
+   - 16GB eMMC
+   - MicroSD up to 256GB
+   - Ethernet 10/100/1000 Native Gigabit
+   - WiFi 802.11a/b/g/n/ac, Bluetooth
+   - USB 3.0 Type C
+   - 2x Video: 4-Lane MIPI CSI (FPV Camera) and 4-Lane MIPI CSI with HMDI Input (Payload Camera)
+
+- **LTE/5G Connectivity Module**
+
+   - Up to 600 Mbps bandwidth
+   - 5G sub-6 and mmWave, SA and NSA operations
+   - 4G Cat 20, up to 7xCA, 256-QAM DL/UL, 2xCA UL
+   - 4 x 4 MIMO for 4G and 5G (sub-6 bands)
+   - 3G HSPA+
+   - Approved by JRL/JTBL, FCC, PTCRB, RED, GCF
+   - Antenna, 4x4 MIMO
+   - Bands: Worldwide
+
+## Set content
```

Comment:
Would be good to improve this title - it sounds like an instruction, but I think you mean "this is what is you get when you buy it"?

Is this the "Airlink Enterprise", or do you get the same thing in both?

Reply:
Right, this is the description of the kit accessories. Both editions have the same kit accessories, the only difference is the enclosure of the unit. Now changed to "Kit Accessories".